org.scilab.Scilab.yaml

```
@@ -0,0 +1,74 @@
+app-id: org.scilab.Scilab
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: scilab
+rename-desktop-file: scilab.desktop
+rename-icon: scilab
+rename-appdata-file: scilab.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+cleanup:
+  - /include
+  - "*.la"
+  - "*.a"
+
+modules:
+  - name: ncurses
+    no-autogen: true
+    config-opts:
+      - --prefix=${FLATPAK_DEST}
```

```suggestion
      - --prefix=${FLATPAK_DEST}
```

You shouldn't need that: it's taken care of